web/api/netdata-swagger.json

```diff
@@ -453,7 +453,35 @@
                 "ses",
                 "des",
                 "cv",
-                "countif"
```

Comment:
@hugovalente-pm hey - any idea if, once this merged, we will need to do some netdata cloud BE work to have these new aggregation functions available in here? 

just wondering as im not sure either way. 

![image](https://user-images.githubusercontent.com/2178292/182938033-220814ee-d431-4270-9f40-2e0e5150d90d.png)

Reply:
I don't we will need work on the BE -unless there is some enum limiting the list @netdata/cloud-be to confirm- but I think we will on FE, maybe @burbuli8ra can confirm on FE I think she worked on this

Reply:
Cool, I'll see how it goes. That drop dowm might get big now so assuming hopefully it will just have a scrollbar as needed maybe. I'll spin up a node on this branch today to test maybe.

Reply:
Hey @hugovalente-pm !
If I understand correctly we need to add 2 new options on the list shown for time aggregation with the values `percentile` and `trimmed_mean`! If that's the case and those 2 new options are not aliases for already existing values, then yes FE work is needed as well!
We will need to add [here](https://github.com/netdata/charts/blob/f7cdf5e0b597cf78c97063b879e95285a664c041/src/components/line/filterToolbox/timeAggregation.js#L6) the new options and we will also need from you to provide as a `label` (shown inside the dropdown menu) and a `short` label (shown when the dropdown is collapsed and the option selected).

Reply:
@burbuli8ra cool good to know. I'll test it a bit on agent end and then follow up with a little PR or issues to have them (or a subset first) available in cloud.

Reply:
@hugovalente-pm I think we need 3 options:

1. `trimmed-mean 5%`
2. `trimmed-median 5%`
3. `percentile 95%`

It would also be interesting to have a UI way to select other percentiles, at least the predefined ones (1, 2, 3, 5, 10, 15, 20, 25 for the trimmed ones, 25, 50, 75, 80, 90, 95, 97, 98, 99 for percentile).